Many tests using Float16 on CPU fail with reference_in_float=False
See https://github.com/pytorch/pytorch/issues/169809

E.g.:
> TestInductorOpInfoCPU.test_comprehensive_grid_sampler_2d_cpu_float16
> [...]
> Mismatched elements: 125 / 780 (16.0%)
> Greatest absolute difference: 0.02001953125 at index (0, 1, 3, 2) (up to 1e-05 allowed)
> Greatest relative difference: 2.34375 at index (1, 1, 2, 4) (up to 0.001 allowed)

Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/test_torchinductor_opinfo.py b/test/inductor/test_torchinductor_opinfo.py
index 807ccb48a79..7e5740e0177 100644
--- a/test/inductor/test_torchinductor_opinfo.py
+++ b/test/inductor/test_torchinductor_opinfo.py
@@ -1329,8 +1329,10 @@ class TestInductorOpInfo(TestCase):
                         # Triton
                         if has_triton():
                             adjusted_kwargs.update(
-                                copy_to_gpu=False, reference_in_float=False
+                                copy_to_gpu=False,
                             )
+                            if device_type == GPU_TYPE:
+                                adjusted_kwargs['reference_in_float'] = False
 
                         # skip checking gradient on CPU for now
                         if device_type == GPU_TYPE:
